core/domain/proto/training_job_response_payload.proto

```diff
@@ -0,0 +1,39 @@
+// coding: utf-8
+//
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "core/domain/proto/text_classifier.proto";
+
+message TrainingJobResponsePayload {
+  message JobResult {
```

Comment:
Please also add documentation comments for this & the payload protos to provide some context for what the proto conceptually contains (which also helps define what the proto should contain over time).

Reply:
Added more comments.